React/React.xcodeproj/project.pbxproj

```diff
@@ -999,6 +999,9 @@
 		66CD94B71F1045E700CB3C7C /* RCTMaskedViewManager.m in Sources */ = {isa = PBXBuildFile; fileRef = 66CD94B01F1045E700CB3C7C /* RCTMaskedViewManager.m */; };
 		66CD94B81F1045E700CB3C7C /* RCTMaskedViewManager.m in Sources */ = {isa = PBXBuildFile; fileRef = 66CD94B01F1045E700CB3C7C /* RCTMaskedViewManager.m */; };
 		68EFE4EE1CF6EB3900A1DE13 /* RCTBundleURLProvider.m in Sources */ = {isa = PBXBuildFile; fileRef = 68EFE4ED1CF6EB3900A1DE13 /* RCTBundleURLProvider.m */; };
+		6E0C2F36229872CD000E3701 /* RCTCxxConvert.h in Copy Headers */ = {isa = PBXBuildFile; fileRef = C60128A91F3D1258009DF9FF /* RCTCxxConvert.h */; };
```

Comment:
What are these files hanging around in React.xcodeproj still? Are they required by TM? could we move them to TM instead? It appears they are just headers, so perhaps we should add their locations to RCTTurboModule's header search paths instead if we need them.

Reply:
Ah, I went looking around and saw that you just made them public. makes sense!

Reply:
Yep, TM imports `<React/RCTCxxConvert.h>`, so it needs to be copied over to `include/React`